### ISSUE TITLE
When will the "reduce" argument in Loss function be released?

### ISSUE THREAD

**Issue by windweller** [pytorch/pytorch#5651]
- OS: MacOS
- PyTorch version: torch-0.3.1
- How you installed PyTorch (conda, pip, source): pip
- Python version: 2.7
- CUDA/cuDNN version: None
- GPU models and configuration: None
- GCC version (if compiling from source): NA

It's not really a bug. I just notice that in the actual code in the repository, there is an additional argument `reduce` for the loss function. For example with `BCELossWithLogits` class, the signature is:
`def __init__(self, weight=None, size_average=True, reduce=True):`

However, in the released version 0.3.1, this change has not ended up in the released version yet. May I ask when will this change happen in the official release?

--------

I solved my problem by copying and pasting code from the current github repo :) but would love to see the changes ending up in the official release.

**Comment by zou3519** [comment]
The change in BCELossWithLogits will be available on the next release. We're (by we I mean mostly @li-roy) have been working on adding the reduce keyword to losses and all of them should be done by the next release. Please refer to #264.

**Comment by windweller** [comment]
Cool, thanks!!
